### PR TITLE
Rename providers section on DataHub user guide 

### PR DESCRIPTION
**Pull request by andrea-manzi**
# Summary 
The Providers section on the DataHub user guide has been renamed to Use Cases as it provides   a very nice description of the different use cases and deployment scenarios

----

**Related issue :**

### CODE REVIEW

File: content/providers/datahub/oneprovider/_index.md [hunk 1]
```diff
@@ -62,22 +62,23 @@ After the deployment is complete, the following message will be shown, with conn
 
 ![image](onedatify_step_5.png)
 
-The WebUI Admin portal at port 9443 can be used to administer the OneProvider installation.
```
Comment (gwarf) [inline content/providers/datahub/oneprovider/_index.md:65]: Why?

File: content/users/datahub/usecases/_index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Use Cases"
+linkTitle: "Use Cases"
+type: docs
+description: "EGI DataHub Use Cases"
+weight: 20
+---
+
+An overview of the use cases and possible deployment scenarios of the EGI DataHub
```
Comment (gwarf) [inline content/users/datahub/usecases/_index.md:9]:
```suggestion
An overview of the use cases and possible deployment scenarios of the EGI DataHub.
```

File: content/users/datahub/usecases/_index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+- PID minting
+- Publishing, discovery and access to datasets
+
+## Integrating DataHub and EGI Notebooks
+
+![image](datahub-notebooks-integration.png)
+
+
+
```
Comment (gwarf) [inline content/users/datahub/usecases/_index.md:49]:
```suggestion
```

**Comment by andrea-manzi** [comment]
ok done!

**Pull request merged**